### PR TITLE
Select default layout model on different platform

### PR DESCRIPTION
**Pull request by fujunwei**

### CODE REVIEW

**Comment by fujunwei** [comment]
@ibelem @Honry @huningxin PTAL, thanks.

File: face_recognition/main.js [hunk 1]
```diff
@@ -50,14 +50,7 @@ $(document).ready(async () => {
 });
 
 $('#backendBtns .btn').on('change', async (e) => {
-  if (inputType === 'camera') utils.stopCameraStream(rafReq, stream);
```
Comment (Honry) [inline face_recognition/main.js:53]: Necessary line is removed.

**Review by Honry — COMMENTED** [review]
Thanks! LGTM % a nit.

**Comment by ibelem** [comment]
@fujunwei do you have any updates?

File: face_recognition/main.js (hunk 1 above)
Comment (Honry) [reply face_recognition/main.js:53]: @fujunwei, friendly ping.

**Comment by fujunwei** [comment]
Sorry for missing your comments that are done in latest code, please take a look again, thanks.

**Review by Honry — APPROVED** [review]

**Pull request merged**